src/keybinding/mod.rs

```diff
@@ -2,6 +2,7 @@ mod bash_zsh;
 mod nushell;
 mod parse;
 mod shell_binding;
+mod fish;
```

Comment:
Sort alphabetically (`cargo fmt` should do that for you)

```suggestion
mod fish;
mod nushell;
mod parse;
mod shell_binding;
```